libs/sdk-ui-kit/src/Timepicker/typings.ts

```diff
@@ -0,0 +1,16 @@
+// (C) 2020 GoodData Corporation
+
+/**
+ * @internal
+ */
+
+export type OverlayPositionType = "absolute" | "fixed" | "sameAsTarget";
```

Comment:
I think this should not be defined here as it is related to Overlay, not TimePicker. We should probably move it to root typings for now (until Overlay is migrated here as well).

Reply:
Or even better to typings/overlay.ts?

Reply:
Moved to `typyings/overlay.ts`